src/plugins/irc/irc-channel.c

```diff
@@ -780,30 +780,18 @@ irc_channel_check_whox (struct t_irc_server *server,
 {
     if ((channel->type == IRC_CHANNEL_TYPE_CHANNEL) && channel->nicks)
     {
-        if (weechat_hashtable_has_key (server->cap_list, "away-notify")
-            || weechat_hashtable_has_key (server->cap_list, "account-notify")
-            || ((IRC_SERVER_OPTION_INTEGER(server, IRC_SERVER_OPTION_AWAY_CHECK) > 0)
-                && ((IRC_SERVER_OPTION_INTEGER(server, IRC_SERVER_OPTION_AWAY_CHECK_MAX_NICKS) == 0)
```

Comment:
Hello,
You are removing the use of options away_check and away_check_max_nicks (this one becomes completely unused in IRC plugin), is that wanted?

Reply:
yes, I don't think there's a reason to ever not use WHOX when it is available

Reply:
But these options are there to let the user enable or disable WHO (or WHOX), based on how many nicks are in the channel. Purpose is to limit the bandwith used (WHO responses can be huge on some channels).

Using WHOX when available is OK for me, but I would like to keep the feature to completely disable WHO(X) if needed.

Reply:
fair enough. i'll tweak the PR in a bit